package.json

```diff
@@ -10,6 +10,7 @@
   },
   "scripts": {
     "test": "NODE_ENV=dev CORS_ORIGIN=http://foo,http://bar scripts/test-local.sh && grunt",
+    "shrinkwrap": "rm -rf node_modules; npm install; npm prune; npm shrinkwrap --dev",
```

Comment:
1. You have to remove ./node_modules and ./npm-shrinkwrap.json
2. Probably better to `&&` chain rm-rf/install/prune/shrinkwrap so it stops at the first failure.